actioncable/lib/action_cable/helpers/action_cable_helper.rb

```diff
@@ -20,9 +20,13 @@ module ActionCableHelper
       # Make sure to specify the correct server location in each of your environments
       # config file:
       #
-      #   config.action_cable.url = "ws://example.com:28080"
+      #   config.action_cable.mount_path = "/cable123"
+      #   <%= action_cable_meta_tag %> would render:
+      #   => <meta name="action-cable-url" content="/cable123" />
+      #
+      #
       def action_cable_meta_tag
-        tag "meta", name: "action-cable-url", content: Rails.application.config.action_cable.url
+        tag "meta", name: "action-cable-url", content: (Rails.application.config.action_cable.mount_path || raise("No Action Cable URL configured"))
```

Comment:
Since we are here can we remove this coupling between Action Cable and Railtie? To do it inject this value in the `ActionCable::Configuration` class in the `ActionCable::Engine`.